en/CommandLine.md

```diff
@@ -1,77 +1,174 @@
+## Basics
 
-## Help: -h
+java -jar JabRef.jar [OPTIONS] [BIBTEX_FILE]
 
-This option causes JabRef to display a summary of the command line options, and quit immediately.
+*Note:* you may have to replace `JabRef.jar` by the actual name of the `.jar` file. 
 
-## No-GUI mode: -n
+You can always specify one or more BibTeX files to load by simply listing their filenames.
+Take care to specify all options before your list of file names.
+You must make sure that the first file name is not misunderstood as being an argument for an option; this simply means that if a boolean option like `-n` or `-l` immediately precedes a file name, add the word `true` as an argument.
+For instance, the command line:
 
-This option suppresses both the JabRef window and the splash window normally shown at startup. It causes the program to quit immediately once the command line options have been processed.
+`java -jar JabRef.jar -o filetoexport.xml,docbook -n true original.bib`
 
+will correctly load the file `original.bib`, export it in docbook format to `filetoexport.xml`, and suppress the GUI.
+The word *true* prevents the file name from being interpreted as an argument to the `-n` option.
+
+
+## Options
+
+`[-a <FILE>] [-asfl] [-b] [-d <FILE>] [--debug] [-f <FILE>] [-g]
+       [-h] [-i <FILE>] [--importToOpen <FILE>] [-m <FILE>] [-n] [-o
+       <FILE>] [-p <FILE>] [-v] [-x <FILE>]`
+
+### Help: `-h`
+(or `--help`)
+
+Displays a summary of the command line options (including the list of available import and export formats), and quit immediately.
+
+
+### No-GUI mode: `-n`
+(or `--nogui`)
+
+Suppresses the JabRef window (i.e. no GUI - Graphic User Interface - is displayed).
+
+It causes the program to quit immediately once the command line options have been processed.
 This option is useful for performing file conversion operations from the command line or a script.
 
-## Import file: -i filename\[,format\]
 
-This option causes JabRef to import or load the specified file. If only a filename is specified, or the filename is followed by a comma and a \* character, JabRef will attempt to detect the file format automatically. This should work for all BibTeX files and files in any import format supported by JabRef. If the filename is followed by a comma and the name of an import format, the given import filter will be used. Use the `-h` option to get a list of availiable import formats.
+### Import file: `-i filename[,import format]`
+(or `--import filename[,import format]` or `--importToOpen filename[,import format]`)
+
+Import or load the file `filename`.
+
+If only the filename is specified (or if the filename is followed by a comma and a `*` character), JabRef will attempt to detect the file format automatically.
+This should work for all BibTeX files, and also for files in any import format supported by JabRef.
```

Comment:
"Should work" -> This works.... and also for all files in a supported input format ....